pyproject.toml

```diff
@@ -69,6 +69,10 @@ scipy = [
 ]
 llama-index = "^0.10.55"
 pytest-asyncio = "^0.23.8"
+pygraphviz = [
+    {version = "^1.13.0", python = ">=3.10,<4.0.0"},
+    {version = "^1.0.0", python = "<3.10"}
+]
```

Comment:
Not sure this is really a dev dependency. Why not put it in extras?

Reply:
It's listed a dev dependencies because they are needed to run the unit tests. It will fail if not added here.